scripts/addFulfillManifest.py

```diff
@@ -43,7 +43,12 @@ def update_batch(metadataObject, bucketName, currKey):
     metadataJSON = json.loads(metadataObject['Body'].read().decode("utf-8"))
     metadataJSONCopy = copy.deepcopy(metadataJSON)
 
-    counter = 0
+    #Checking if record asscociated with manifest is public domain or in copyright
```

Comment:
what's the difference between the script and the process? is the script just supposed to be run locally? 

If there is duplicate logic, then I think we should encapsulate it in one place.

Reply:
The script was initially made as a manual process to add the fulfill manifests which was later used to create the fulfill process which is an automatic task that can be run manually. We don't need the script anymore since the process will serve both purposes.

Reply:
should we delete the script?

Reply:
Yes I believe we should and in my newest commit I did delete the script.